rfd/0001-tooling-requirements.md

```diff
@@ -0,0 +1,150 @@
+---
+authors: Fred Heinecke (fred.heinecke@goteleport.com)
+state: draft
+---
+
+# RFD 1 - Tooling Requirements
+
+## Required approvers
+
+* Engineering: (@r0mant)
```

Comment:
I am not quite sure who should be listed here. Does any reviewers have suggestions?

Reply:
You can add `* Security: (@reedloden  || @wadells)`.  I don't know that product will have an opinion here.  Otherwise, I think you've added the right reviwers already.